students/WWhite/lesson08/circle.py

```diff
@@ -0,0 +1,108 @@
+import math
+
+
+class Circle(object):
+    def __init__(self, radius):
+        if radius >=0 :
+            self.radius = radius
+        else:
+            raise TypeError("Invalid radius passed")
+
+    @classmethod
+    def from_diameter(class_object, diameter):
+        if diameter >= 0:
+            return class_object(diameter / 2)
+        else:
+            raise TypeError("Invalid diameter passed")
+
+    @property
+    def radius(self):
+        return self._radius
```

Comment:
you don't need to use a `@property` or setter if you aren't adding any special logic or calculations at the time of attribute assignment or access